extensions/widgets/svgpath/svgpath.lcb

```diff
@@ -287,7 +287,7 @@ public handler OnLoad(in pProperties as Array)
 	put pProperties["maintain aspect ratio"] into mMaintainAspectRatio
 	put pProperties["flip vertically"] into mFlipVertically
 	put pProperties["angle"] into mAngle
-
+	put pProperties["iconPresetName"] into mPathName
```

Comment:
You should put this in a `if "iconPresetName" is among the keys of pProperties` block so that existing saved instances of SVG widget do not error when they load.